### PR TITLE
adding disclaimer for coreutils

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Adding a disclaimer for curl code examples

Adding ruby and python code snippets for Tracing API 

### Motivation

Some people were disturbed by curl example not working on MacOsX and code snippets missing

### Preview link

https://docs-staging.datadoghq.com/gus/api-disclaimer/api/?lang=python#overview

### CODE REVIEW

**Comment by irabinovitch** [comment]
I don't think this is the right approach.  Lets write up python and ruby examples as I suggested in https://github.com/DataDog/documentation/issues/1705

File: content/api/overview/overview.md [hunk 1]
```diff
@@ -6,4 +6,6 @@ order: 1
 ## Overview
 This section details Datadogs HTTP API. It makes it easy to get data in and out of Datadog.
 
-The Datadog API uses resource-oriented URLs, uses status codes to indicate the success or failure of requests and returns JSON from all requests. Let's dive in and see how it works.
\ No newline at end of file
+The Datadog API uses resource-oriented URLs, uses status codes to indicate the success or failure of requests and returns JSON from all requests. Let's dive in and see how it works.
+
+**Note**: Curl code examples displayed work only with `coreutils`, if you are working on a MacOsX device, install it with `brew install coreutils`
```
Comment (jyee) [inline content/api/overview/overview.md]: cURL code examples are intended for use on Linux systems. To run the examples on a Mac, you will need to install `coreutils` using the [homebrew package manager](https://brew.sh/): `brew install coreutils`

Comment (irabinovitch) [reply content/api/overview/overview.md]: Maybe:
cURL code examples assume usage of BASH and GNU coreutils. On OSX you can install coreutils via the Homebrew package manager: brew install coreutils

File: content/api/overview/overview.md [hunk 2]
```diff
@@ -6,4 +6,6 @@ order: 1
 ## Overview
 This section details Datadogs HTTP API. It makes it easy to get data in and out of Datadog.
 
-The Datadog API uses resource-oriented URLs, uses status codes to indicate the success or failure of requests and returns JSON from all requests. Let's dive in and see how it works.
\ No newline at end of file
+The Datadog API uses resource-oriented URLs, uses status codes to indicate the success or failure of requests and returns JSON from all requests. Let's dive in and see how it works.
+
+**Note**: cURL code examples assume usage of BASH and GNU coreutils. On OSX you can install coreutils via the Homebrew package manager: `brew install coreutils`
```
Comment (jyee) [inline content/api/overview/overview.md]: Let's make "Homebrew package manager" a link to https://brew.sh/

File: content/api/tracing/code_snippets/send_trace.sh [hunk 3]
```diff
@@ -11,7 +11,7 @@ START=$(date +%s%N)
 sleep 2
 
 # Stop the timer.
-DURATION=$(($(date +%s%N) - $START))
+DURATION= $(($(date +%s%N) - $START))
```
Comment (jyee) [inline content/api/tracing/code_snippets/send_trace.sh]: Need to revert this. The space after the = is not allowed and results in a `command not found` error.

File: content/api/tracing/code_snippets/send_trace.rb [hunk 4]
```diff
@@ -1,29 +1,26 @@
...
+DURATION = ((Time.now.to_f - START)* 1000000).to_i
+
+# Send the traces.
+port = 8126
+host = "127.0.0.1"
+path = "/v0.3/traces"
+
+req = Net::HTTP::Put.new(path, initheader = { 'Content-Type' => 'application/json'})
+
+req.body = { "trace_id": TRACE_ID,"span_id": SPAN_ID,"name": "span_name","resource": "/home","service": "service_name","type": "web","start": START,"duration": DURATION}
```
Comment (jyee) [inline content/api/tracing/code_snippets/send_trace.rb]: let's split this into multiple lines so it reads easier.

File: content/api/tracing/code_snippets/send_trace.py [hunk 5]
```diff
@@ -1,29 +1,30 @@
...
+	"trace_id": TRACE_ID,\
+	"span_id": SPAN_ID,\
+	"name": "span_name",\
+	"resource": "/home",\
+	"service": "service_name",\
+	"type": "web",\
+	"start": START,\
+	"duration": DURATION}
+	
+requests.put('http://localhost:8126/v0.3/traces', data = data, headers=headers)
```
Comment (jyee) [inline content/api/tracing/code_snippets/send_trace.py]: Let's ensure we have consistent python code styling with spaces around equals and lowercase var names e.g. `span_id = random...`

File: content/api/overview/overview.md (hunk 2 above)
Comment (l0k0ms) [reply content/api/overview/overview.md]: 👍

File: content/api/tracing/code_snippets/send_trace.sh (hunk 3 above)
Comment (l0k0ms) [reply content/api/tracing/code_snippets/send_trace.sh]: 👍

File: content/api/tracing/code_snippets/send_trace.rb (hunk 4 above)
Comment (l0k0ms) [reply content/api/tracing/code_snippets/send_trace.rb]: 👍

File: content/api/tracing/code_snippets/send_trace.py (hunk 5 above)
Comment (l0k0ms) [reply content/api/tracing/code_snippets/send_trace.py]: creating a new card for this.

**Pull request merged**